pipelines/upgrade_pipeline.yml

```diff
@@ -130,7 +132,8 @@
       scenario: "{{ pipeline_type }}"
       scenario_version: "{{ forklift_upgrade_version_intermediate }}"
     - foreman_server_repositories
-    - foreman_client_repositories
+    - role: foreman_client_repositories
+      when: pipeline_type != 'foreman'
```

Comment:
Would it make sense to move this down into the testing section given the client repositories are only used for that aspect? I agree that this check still needs to be here but it more localizes the use of these.

Reply:
Yeah, I thought about that and then put it off as "don't do two changes in one go", but if you also thought that, I'll go ahead and do that.

Reply:
done